ethercat-sys/build.rs

```diff
@@ -33,40 +30,59 @@ fn main() {
         .expect("Couldn't write bindings!");
 
     // Generate the EC_IOCTL_ ioctl numbers -- bindgen can't handle them.
-    let code = fs::read_to_string(&format!("{}/master/ioctl.h", path))
-        .expect("master/ioctl.h not found");
+    let code =
+        fs::read_to_string(&format!("{}/master/ioctl.h", path)).expect("master/ioctl.h not found");
     let mut new = String::new();
     for line in code.split('\n') {
         let parts = line.split_whitespace().collect::<Vec<_>>();
-        if parts.len() >= 3 &&
-            parts[0] == "#define" &&
-            parts[1].starts_with("EC_IOCTL_") &&
-            parts[2].starts_with("EC_IO")
+        if parts.len() >= 3
+            && parts[0] == "#define"
+            && parts[1].starts_with("EC_IOCTL_")
+            && parts[2].starts_with("EC_IO")
         {
             let name = &parts[1]["EC_IOCTL_".len()..];
+
+            // FIXME:
+            // There are constant names that start with a number
+            // so they would be invalid rust code.
+            // The problem exists e.g. if you try to
+            // compile the v1.5.2-sncn-11 branch of the synapticon
+            // fork: https://github.com/synapticon/Etherlab_EtherCAT_Master
+            // So as a first dirty workaround we just ignore them
+            // to be able to compile.
+            if name.starts_with("64_REF_CLK") {
+                continue;
+            }
+
             let mut numparts = parts[2].split("(");
             let access = match numparts.next().unwrap() {
                 "EC_IO" => match name {
                     "SEND" => "arg",
```

Comment:
I noticed some more `arg` in your port for sncn-11, do you want to add some?

Reply:
Good point! I'll hopefully have a look at it tomorrow :)